app/(site)/about/page.tsx

```diff
@@ -0,0 +1,31 @@
+import { Metadata } from "next";
+import AvatarLineup from "./AvatarLineup";
+import SectionHeader from "@/components/Common/SectionHeader";
+import MenuDescription from "./MenuDescription";
+
+export const metadata: Metadata = {
+  title: "About",
+  description: "",
+};
+
+const AboutPage = async () => {
+  return (
+    <section className="py-20 lg:py-25 xl:py-30">
+        <div className="mx-auto max-w-c-1315 px-4 md:px-8 xl:px-0">
+        <SectionHeader
```

Comment:
This could be in the AvatarLineup Component, because is directly associated with it.